test/new_huffman_tb.py

```diff
@@ -0,0 +1,82 @@
+# !/usr/bin/env python3
+# This is the module for testing the Huffman Encoder.
+
+from litex.gen import *
+
+from litex.soc.interconnect.stream import *
+from litex.soc.interconnect.stream_sim import *
+
+from litejpeg.core.common import *
+from litejpeg.core.huffman.huffmancore import HuffmanEncoder
+
+from common import *
+
+"""
+Testbanch for the Huffman module.
```

Comment:
`Testbanch.`

Can you run spell check on all the comments? What do you use for an editor?

Reply:
I am using sublime as my text editor, working on that.